clang/lib/APINotes/APINotesReader.cpp

```diff
@@ -624,6 +624,13 @@ class TagTableInfo
                                         ReleaseOpLength - 1);
       Data += ReleaseOpLength - 1;
     }
+    unsigned DefaultOwnershipLength =
+        endian::readNext<uint16_t, llvm::endianness::little>(Data);
+    if (DefaultOwnershipLength > 0) {
```

Comment:
```suggestion
    if (unsigned DefaultOwnershipLength =
            endian::readNext<uint16_t, llvm::endianness::little>(Data)) {
```

This matches the style in the rest of the code.

Reply:
For `ConformanceLength`, yes, but for `ImportAsLength`, `RetainOpLength`, and `ReleaseOpLength`, we'll stick to the current style. I'm keeping this style because this change is more closely tied to `SwiftImportAs` rather than `SwiftConformance`.